src/test/java/org/cactoos/io/ResourceOfTest.java

```diff
@@ -63,15 +64,30 @@ public void readsBinaryResource() throws Exception {
     }
 
     @Test
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@yegor256 why do we need this at the method level? if it's needed let's just use it once at class level for consistency